### DISCUSSION TITLE
Why it is "not supported for your OS"? RasPi 4 with latest HomeAssistant

### DISCUSSION THREAD

**Discussion by iluhis** [hassio-addons/addon-mopidy#130] [Q&A]
Hello, Strangely discovered that Mopidy is not available on my system, it shows "This add-on is not compatible with the processor or operating system of your device.". I have RasPi4 with latest for today HomeAssistant onboard. About a year ago I was able to install it and it worked.

**Comment by frenck** [comment]
Support for that system has been dropped by this add-on quite some time ago.
